klog.go

```diff
@@ -424,10 +426,12 @@ func InitFlags(flagset *flag.FlagSet) {
 	}
 
 	flagset.StringVar(&logging.logDir, "log_dir", logging.logDir, "If non-empty, write log files in this directory")
-	flagset.StringVar(&logging.logFile, "log_file", logging.logFile, "If non-empty, use this log file")
+	flagset.StringVar(&logging.logFile, "log_file", logging.logFile, "If non-empty, use this log file (mutually exclusive with the log-dir option)")
```

Comment:
should it error out if both flags are specified?

Reply:
These two flags have a priority originally, if the `log_file` is set, then the `log_dir` will not take effect